Plugins/Util/NWScript/nwnx_util.nss

```diff
@@ -18,6 +30,8 @@ effect NWNX_Util_ItemPropertyToEffect(itemproperty ip);
 string NWNX_Util_GenerateUUID();
 // Strip any color codes from a string
 string NWNX_Util_StripColors(string str);
+// Is a valid resref
+int NWNX_Util_IsValidResRef(string filename, int type=NWNX_UTIL_RESREF_TYPE_CREATURE);
```

Comment:
Nitpick: I'd use `resref` instead of `filename`

Even smaller nitpick: spaces before and after the `=` for type :D

Reply:
I copied how the default was done in `NWNX_Util_GetCurrentScriptName(int depth=0)` so blame sherincall (fixed that one too)